### ISSUE TITLE
App crashes on start-up

### ISSUE THREAD

**Issue by bmac1993** [expo/expo#113]
Just migrated my app to use Expo. Was able to get a couple of errors which I fixed and now when my app loads, both through the iOS simulator and on a device, the app immediately crashes. Nothing is in the console or being logged. Any specific code you'd like to see?

**Comment by bmac1993** [comment]
https://cloud.githubusercontent.com/assets/7414766/25066911/a84be470-2201-11e7-9dfb-76d2e1046464.png

**Comment by bmac1993** [comment]
Looks like I found the issue, may have been with migration. I *think * my expo node_modules were referencing v15.0 & my package.json referencing sdk v14.0. Followed the steps to update, however my exp.json was still at v14. 

I would just check the migration to ensure everything is installing the latest version since I installed both the SDK and did the migration today & had inconsistencies. Then again, I may have completely botched/missed something.
